src/hb-subset-plan.cc

```diff
@@ -157,12 +164,11 @@ _closure_glyphs_lookups_features (hb_face_t	     *face,
 
   // Collect and prune features
   hb_set_t feature_indices;
-  _collect_subset_layout (face,
-			  table_tag,
-			  layout_features_to_retain,
-			  retain_all_features,
-			  hb_ot_layout_collect_features,
-			  &feature_indices);
+  _collect_subset_layout<T> (face,
```

Comment:
This name, `_collect_subset_layout` means nothing. What is this doing?

Reply:
Renamed to _collect_layout_indices which better indicates what it does. It runs the provided function to collect a type of indices from a layout table.